frontend/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import AppRouter from './router'
+import AppRouter from './router.js'
 
 export default function App() {
   return (
```

Comment:
When you done moving routing to index. then this file can be deleted

Reply:
✔️